examples/rust/hello/hello/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
```

Comment:
why not put into examples/rust/hello/Cargo.toml

Reply:
The outer "hello" directory serves as the application entry point on the NuttX side, while the inner "hello" contains pure Rust application logic. Their relationship is similar to that of lvgl and other third-party libraries, and integrating them together is entirely feasible.

Reply:
but it is a demo in source, not 3rd library. only 3rd package need add the intermediate folder

Reply:
OK